### PR TITLE
validate_request for RefreshTokenGrantType was rewritten in general form

### PR DESCRIPTION
**Pull request by aliev**

### CODE REVIEW

**Comment by aliev** [comment]
cc @danygielow

File: tests/test_grant_type.py [hunk 1]
```diff
@@ -92,10 +91,7 @@ async def test_refresh_token_grant_type(
...
     # Check that previous token was revoken
     token_in_db = await db.get_token(
-        request, client_id, old_token.access_token, old_token.refresh_token
+        request, client_id, defaults.access_token, defaults.refresh_token
     )
     assert token_in_db.revoked
     assert token_response.scope == "read"
-
-    with pytest.raises(InvalidGrantError):
-        await grant_type.validate_request(request)
```
Comment (danygielow) [inline tests/test_grant_type.py:101]: This should still be tested. A test for expired refresh tokens should also be added.

**Review by danygielow — COMMENTED** [review]
Tests for revoked tokens (was removed) and expired refresh tokens (new) would be nice.
See Review.

**Comment by aliev** [comment]
@danygielow good point! I will fix this evening :)

File: tests/test_grant_type.py (hunk 1 above)
Comment (aliev) [reply tests/test_grant_type.py:101]: @danygielow updated

**Review by danygielow — APPROVED** [review]

File: tests/test_grant_type.py (hunk 1 above)
Comment (aliev) [reply tests/test_grant_type.py:101]: regarding of the expired refresh token we already have a tests of it:

https://github.com/aliev/aioauth/blob/69649d7f91f1416eca7804b360de4f9a0dfd282f/tests/test_request_validator.py#L215-L235

**Pull request merged**